include/mbedtls/mbedtls_config.h

```diff
@@ -3116,7 +3116,10 @@
  *          library/x509_csr.c
  *
  * Requires: MBEDTLS_ASN1_PARSE_C, MBEDTLS_BIGNUM_C, MBEDTLS_OID_C,
- *           MBEDTLS_PK_PARSE_C
+ *           (MBEDTLS_MD_C or MBEDTLS_USE_PSA_CRYPTO)
+ *
+ * \warning If building without MBEDTLS_MD_C, you must call psa_crypto_init()
+ * before doing any PKCS#1 v2.1 operation.
```

Comment:
I'm not sure if that's needed. Are there any hash operations performed only with `MBEDTLS_X509_USE_C` that would require `psa_crypto_init()`, or is it just metadata that gets accessed? Also (if it's needed):
```suggestion
 * before doing any x509 operation.
```

Reply:
This was requirement from the task:

> In check_config.h, declare both MBEDTLS_X509_USE_C and MBEDTLS_X509_CREATE_C as depending on MBEDTLS_MD_C || MBEDTLS_USE_PSA_CRYPTO. (Note: they have no explicit dependency on MD right now, that's because the dependency was implicit via PK, but after https://github.com/Mbed-TLS/mbedtls/pull/6065 PK will no longer depend on MD, so we should have an explicit declaration now.) Adjust the description in mbedtls_config.h with a warning (as in https://github.com/Mbed-TLS/mbedtls/pull/6141).

I think the dependency to PK should stay and copy paste error should be fixed.

Reply:
I agree that this warning can be removed as there are no such dependencies at least I con't see any.

Reply:
I think that both case for `MBEDTLS_X509_USE_C and MBEDTLS_X509_CREATE_C` should be clarified by @mpg.